.github/workflows/typespec-validation-all.yaml

```diff
@@ -0,0 +1,74 @@
+name: TypeSpec Validation All
+
+on:
+  push:
+    branches:
+      - main
+      - RPSaaSMaster
+      - typespec-next
+
+  pull_request:
+    branches:
+      - main
+      - RPSaaSMaster
+      - typespec-next
+    paths:
+      - .gitattributes
+      - .prettierrc.json
+      - package-lock.json
+      - package.json
+      - tsconfig.json
+      - eng/**
+      - specification/suppressions.yaml
+      - specification/common-types/**
+
+      # Workflow and workflow dependencies
+      - .github/workflows/typespec-validation-all.yml
```

Comment:
```suggestion
      - .github/workflows/typespec-validation-all.yaml
```

Reply:
This is going to be an exciting habit to break